torch_geometric/nn/aggr/multi.py

```diff
@@ -0,0 +1,79 @@
+import inspect
+from typing import Any, List, Optional, Union
+
+import torch
+from torch import Tensor
+
+from . import basic, lstm
+from .base import Aggregation
+
+
+def normalize_string(s: str) -> str:
```

Comment:
Do we want to move the `aggregation_resolver` to `nn/resolver.py`? It raises circular import error since I need to import `nn.aggr` to `nn/resolver.py` and import `aggregation_resolver` to `nn.aggr.multi`. Show we make a python class for the resolver so that we don’t need to import the package to `nn/resolver.py` and can instantiate it for different packages?

Reply:
You can import `nn.aggr` inside the `aggregation_resolver()` which should fix this issue.